ui/src/api/generated/base.ts

```diff
@@ -0,0 +1,86 @@
+/* tslint:disable */
```

Comment:
I think we shouldn't commit any of the generated files. If the openapi doesn't change then the output should always be the same and if it does we don't really care about the changes unless it breaks the CI but then an error will be thrown and you won't be able to ship your code until you fix it.